product_dimension/README.rst

```diff
@@ -0,0 +1,48 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Product Dimension
```

Comment:
update readme according to https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst